packages/block-editor/src/components/block-list/block.js

```diff
@@ -644,12 +666,6 @@ function BlockListBlockProvider( props ) {
 				hasEditableOutline:
 					blockEditingMode !== 'disabled' &&
 					getBlockEditingMode( rootClientId ) === 'disabled',
-				className: hasLightBlockWrapper
-					? attributes.className
-					: undefined,
-				defaultClassName: hasLightBlockWrapper
-					? getBlockDefaultClassName( blockName )
-					: undefined,
```

Comment:
I'm surprised this PR doesn't cause more issues. I feel like defensively, we should have default values for all of these keys in the `previewContext` object no?

Reply:
Why all? Only some end up as a public API. The rest is all internal context that we have control over. Fixed the public API ones in 04c36b8e85d4cc26104adfc38482d2fc9513a3d9.

Reply:
It was more about avoiding breakage. Like code base assuming some keys are defined...

Reply:
I can check all keys in a bit. Most of them are just used once in the block props hook, so it should be easy to double check.

Reply:
Added one more (`mode`) since it's a string and also public API. But the rest all end up in a `classnames` call or falsy checks.